src/quo/components/community/channel_actions.cljs

```diff
@@ -1,32 +1,16 @@
 (ns quo.components.community.channel-actions
```

Comment:
can we move this to a view file.
`quo.components.community.channel-actions.view`

Reply:
also it's better to use one style file for each component.
quo.components.community.channel-actions.view
quo.components.community.channel-actions.style 
and
quo.components.community.channel-actions.view
quo.components.community.channel-action.style

for shared styles it's better to do so through encapsulation with components 👍